rotkehlchen/tasks/manager.py

```diff
@@ -224,6 +224,9 @@ def _run_ethereum_txreceipts_query(self, hash_results: List[Tuple]) -> None:
         for entry in hash_results:
             tx_hash = '0x' + entry[0].hex()
             tx_receipt_data = self.chain_manager.ethereum.get_transaction_receipt(tx_hash=tx_hash)
+            # add_receipt_data expects the transactionHash to be a string and
+            # get_transaction_receipt returns bytes for this field.
+            tx_receipt_data['transactionHash'] = tx_receipt_data['transactionHash'].hex()
```

Comment:
Ah wait. If it's a bytes vs strings thing it may be more complex. web3.py (so all our connections except etherscan) return things in one way while the etherscan api returns them in another. The job of the ethereum manager is to unify all these. So this should happen there and a test should exist. If it's this.

Can you give some more details on what happened?

Reply:
I'm a bit lost. At first by checking the logic I believe that my change fixed the thing and tried it and indeed I didn't see the error. When checking the failing test seems that my fix doesn't fix the whole issue and I'm trying to figure out why. I think it's what you mention by what I've tried but I'm on it